hubmap_commons/hm_auth.py

```diff
@@ -129,7 +129,7 @@ def getGroupDisplayName(group_uuid):
         return grps_by_id[group_uuid]['displayname']
 
 
-    def __init__(self, clientId, clientSecret):
+    def __init__(self, clientId, clientSecret, globusGroups):
```

Comment:
Assigning a default value to `globusGroups` like below would make it more obvious that this is an optional argument:
```
def __init__(self, clientId, clientSecret, globusGroups=None):
```

Reply:
@yuanzhou This should be resolved in latest commit